### PR TITLE
AzureSQL redirect

### PR DESCRIPTION
**Pull request by windoze**
Support AzureSQL redirect

### CODE REVIEW

**Comment by windoze** [comment]
Hi @kardeiz, would you like to take a look?
Thanks.

File: src/lib.rs [hunk 1]
```diff
@@ -71,10 +71,33 @@ pub mod rt {
             use tokio_util::compat::TokioAsyncWriteCompatExt;//Tokio02AsyncWriteCompatExt;
 
             let tcp = TcpStream::connect(self.config.get_addr()).await?;
+            tcp.set_nodelay(true)?;
```
Comment (kardeiz) [inline src/lib.rs]: @windoze You can drop this; the `set_nodelay` is performed in the next line `(self.modify_tcp_stream)(&tcp)?;`. Please do so for the line in the `async_std` impl as well.

**Review by kardeiz — CHANGES_REQUESTED** [review]
@windoze Looks good, thanks! I've requested a minor change on two lines.

**Comment by windoze** [comment]
Thanks for your reply, I didn't see it 😄 

Redundant set_nodelay removed.

**Review by kardeiz — APPROVED** [review]
Thanks! I'm merging this and will release a `0.5.2` in a bit.

**Pull request merged**